backend/public-graph/graph/resolver.go

```diff
@@ -2781,12 +2723,29 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 				StackTrace:     &traceString,
 				Timestamp:      v.Timestamp,
 				Payload:        v.Payload,
-				RequestID:      nil,
 				IsBeacon:       isBeacon,
 				ServiceVersion: serviceVersion,
 				ServiceName:    sessionObj.ServiceName,
 			}
 
+			if !r.IsErrorIngested(ctx, projectID, &publicModel.BackendErrorObjectInput{
```

Comment:
I noticed that downstream we call `isIngestedByRateLimit` - will we be double counting errors toward this rate limit? Or were these previously not being counted?

Reply:
these errors (frontend errors) were incorrectly not counted previously / not filtered out. this is the only place frontend errors would go thru the sampling check, so they won't be double counted.